src/main/java/org/folio/circulation/rules/cache/CirculationRulesCache.java

```diff
@@ -62,11 +64,18 @@ public void clearCache(String tenantId) {
     rules.reloadTimestamp = 0;
   }
 
-  private boolean isCurrent(Rules rules) {
+  private boolean isCurrent(String tenantId, Rules rules) {
     if (rules == null) {
+      log.info("Rules object is null considering it not current for tenant {}", tenantId);
       return false;
     }
-    return rules.reloadTimestamp + MAX_AGE_IN_MILLISECONDS > System.currentTimeMillis();
+
+    boolean isCurrent = rules.reloadTimestamp + MAX_AGE_IN_MILLISECONDS >
+      System.currentTimeMillis();
```

Comment:
Should we extract `System.currentTimeMillis()` into a local variable? Otherwise timestamp used to calculate `isCurrent` and the one in the log below will be slightly different?